projects/debian/Dockerfile

```diff
@@ -0,0 +1,37 @@
+ARG BASE_IMAGE="debian"
+ARG BASE_IMAGE_VERSION="jessie-slim"
+ARG VERSION=${BASE_IMAGE_VERSION}
+
+FROM ${BASE_IMAGE}:${BASE_IMAGE_VERSION}
+
+ARG VERSION=${BASE_IMAGE_VERSION}
+
+ARG PACKAGES="software-properties-common"
+
+RUN apt-get update && \
+	apt-get -u -y dist-upgrade
+
+RUN apt-get -y install ${PACKAGES}
```

Comment:
We can reduce the image size doing this all in one layer and cleaning up the apt-cache like this:
```
RUN apt-get update && \
    apt-get -u -y dist-upgrade && \
    apt-get -y --no-install-recommends install ${PACKAGES} && \
    rm -rf /var/lib/apt/lists/*
```
This is just a suggestion and optional, with `--no-install-recommends` we can make the install package footprint a little bit smaller as well.